_includes/charts/calico/templates/calico-kube-controllers.yaml

```diff
@@ -95,11 +104,20 @@ spec:
               value: node
             - name: DATASTORE_TYPE
               value: kubernetes
+          livenessProbe:
+            exec:
+              command:
+              - /usr/bin/check-status
+              - -r
+            periodSeconds: 10
+            initialDelaySeconds: 10
+            failureThreshold: 6
           readinessProbe:
```

Comment:
Is it intentional that this readinessProbe doesn't match the one above? (initialDelaySeconds vs periodSeconds).

Could you comment on why we should make these changes to the readinessprobe as well? The PR seems to mostly be about the liveness probe.

Reply:
You are right, should be `periodSeconds`. Will fix. Thanks.

Reply:
For `periodSeconds`, the default value is `10`. Other probes in the same manifest have this set anyway and wanted to keep the same style.